app/services/SubmissionService.scala

```diff
@@ -25,15 +25,15 @@ import models.ReturnToStartException
 import models.cache._
 import play.api.http.Status
 import repositories.CacheRepository
-import services.audit.{AuditService, AuditTypes}
+import services.audit.{AuditService, AuditType}
 import uk.gov.hmrc.http.HeaderCarrier
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success}
 
 @Singleton
 class SubmissionService @Inject()(
-  cache: CacheRepository,
+  cacheRepository: CacheRepository,
```

Comment:
Every other controller calls this `cache` can we leave it as it is please

Reply:
I've encountered variable hiding bacause of that, so I'd rather go the other way - to rename it to `cacheRepository` in all controllers.

Reply:
Sounds like the variable should be named differently rather than changing every single controller?

Reply:
The other variable is of type `Cache` so it is natural to name it `cache`. In this case it is misleading to give variable of type `CacheRepository` the same name. In my opinion it is more correct to name `CacheRepository` instances with something more meaningful.